### PR TITLE
OCPBUGS-43762: adds day 2 operation for masquerade to docs

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.17+

Issue:
https://issues.redhat.com/browse/OCPBUGS-43762

Link to docs preview:
https://85891--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/configure-ovn-kubernetes-subnets.html#nw-ovn-k-day-2-masq-subnet_configure-ovn-kubernetes-subnets

QE review:
- [x] QE has approved this change.

Additional information:
PTAL @zhaozhanqi

### CODE REVIEW

File: modules/nw-ovn-k-day-2-masq-subnet.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+** For dualstack clusters using IPv6, run the following command:
++
+[source,terminal]
+----
+$ oc patch networks.operator.openshift.io cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"gatewayConfig":{"ipv4":{"internalMasqueradeSubnet": "<masquerade_subnet>"},"ipv6":{"internalMasqueradeSubnet": "<masquerade_subnet>"}}}}}}'
+----
++
+Where:
+
+`internalMasqueradeSubnet`:: Specifies an IP address to be used as the masquerade subnet. This range cannot overlap with any other subnets used by {product-title} or on the host itself. The default values for IPv4 is `100.254.0.0/17` and for IPv6 the default value is
```
Comment (zhaozhanqi) [inline modules/nw-ovn-k-day-2-masq-subnet.adoc:29]: The default value for ipv4 and ipv6 is not correct, see here https://github.com/openshift/ovn-kubernetes/blob/master/go-controller/pkg/config/config.go#L145-L146

File: modules/nw-ovn-k-day-2-masq-subnet.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+`fe69::/112`
+
+** For clusters using IPv4, run the following command:
++
+[source,terminal]
+----
+$ oc patch networks.operator.openshift.io cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"gatewayConfig":{"ipv4":{"internalMasqueradeSubnet": "<masquerade_subnet>"}}}}}}'
+----
++
+`internalMasqueradeSubnet`:: Specifies an IP address to be used as the masquerade subnet. This range cannot overlap with any other subnets used by {product-title} or on the host itself. The default values for IPv4 is `100.254.
```
Comment (zhaozhanqi) [inline modules/nw-ovn-k-day-2-masq-subnet.adoc:39]: for default value https://github.com/openshift/ovn-kubernetes/blob/master/go-controller/pkg/config/config.go#L145-L146

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM!

File: modules/nw-ovn-k-day-2-masq-subnet.adoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+** For dualstack clusters using IPv6, run the following command:
++
+[source,terminal]
+----
+$ oc patch networks.operator.openshift.io cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"gatewayConfig":{"ipv4":{"internalMasqueradeSubnet": "<ipv4_masquerade_subnet>"},"ipv6":{"internalMasqueradeSubnet": "<ipv6_masquerade_subnet>"}}}}}}'
+----
++
+Where:
+
+`ipv6_masquerade_subnet`:: Specifies an IP address to be used as the IPv6 masquerade subnet. This range cannot overlap with any other subnets used by {product-title} or on the host itself. The default value for IPv6 is `fd69::/125`.
```
Comment (stevsmit) [inline modules/nw-ovn-k-day-2-masq-subnet.adoc:29]: ipv4 comes first in the command so I'd suggest switching these around.

File: modules/nw-ovn-k-day-2-masq-subnet.adoc [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+. Change your cluster's masquerade subnet:
+
+** For dualstack clusters using IPv6, run the following command:
++
+[source,terminal]
+----
+$ oc patch networks.operator.openshift.io cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"gatewayConfig":{"ipv4":{"internalMasqueradeSubnet": "<ipv4_masquerade_subnet>"},"ipv6":{"internalMasqueradeSubnet": "<ipv6_masquerade_subnet>"}}}}}}'
+----
++
+Where:
```
Comment (stevsmit) [inline modules/nw-ovn-k-day-2-masq-subnet.adoc:27]: where is most commonly in lowercase when used like this

File: modules/nw-ovn-k-day-2-masq-subnet.adoc [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+`ipv4_masquerade_subnet`:: Specifies an IP address to be used as the IPv4 masquerade subnet. This range cannot overlap with any other subnets used by {product-title} or on the host itself. The default value for IPv4 is `169.254.169.0/29`.
+
+** For clusters using IPv4, run the following command:
++
+[source,terminal]
+----
+$ oc patch networks.operator.openshift.io cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"gatewayConfig":{"ipv4":{"internalMasqueradeSubnet": "<ipv4_masquerade_subnet>"}}}}}}'
+----
++
+Where:
```
Comment (stevsmit) [inline modules/nw-ovn-k-day-2-masq-subnet.adoc:40]: Same here

**Comment by zhaozhanqi** [comment]
@JoeAldinger this feature (day 2 operation for masquerade) is only for 4.17+ version, I saw you backport to versions <=4.16 ?

**Comment by JoeAldinger** [comment]
@zhaozhanqi i have a big that required/requested the backport. https://issues.redhat.com/browse/OCPBUGS-42556

I was going to ping you but forgot. If this needs reverted please let me know. I'll do asap.

**Comment by zhaozhanqi** [comment]
yes,  I think this should be revert for version<=4.16 
cc. @bpickard22

**Pull request merged**